MHVLib/Models/MHVModelBase.m

```diff
@@ -212,6 +212,11 @@ + (void)setProperties:(NSObject<MHVDataModelProtocol> *)object fromDictionary:(N
                             
                             @synchronized (propertyParametersCache)
                             {
+                                if (!objectParameters)
+                                {
+                                    objectParameters = [NSString class];
```

Comment:
hmmm... github filtered my comment, so it doesn't make a lot of sense. I was trying to say I wasn't sure if this would work if the datatype were an NSArray of NSNumber. Same conclusion - I figured we'd revisit that if and when it's needed.